src/legacy/ui/public/vis/editors/default/components/agg_params.tsx

```diff
@@ -147,7 +146,7 @@ function DefaultEditorAggParams({
         onAggParamsChange(agg.params, param, newValue);
       }
     });
-  }, [agg.type]);
+  }, [agg.type, editorConfig]);
```

Comment:
`editorConfig` will change if `agg.type` changes:

```suggestion
  }, [editorConfig]);
```

Reply:
Nice catch. 
Fixed.